draft-ietf-taps-interface.md

```diff
@@ -303,7 +303,7 @@ connection-oriented. Connections can be created from Preconnections in three way
 
 - by initiating the Preconnection (i.e., actively opening, as in a client; {{initiate}}),
 - through listening on the Preconnection (i.e., passively opening, as in a server; {{listen}}),
-- or rendezvousing on the Preconnection (i.e., peer to peer establishment; {{rendezvous}}).
+- or a rendezvous for the Preconnection (i.e., peer to peer establishment; {{rendezvous}}).
```

Comment:
```suggestion
- or via a rendezvous for the Preconnection (i.e., peer to peer establishment; {{rendezvous}}).
```

I think this needs a word such as "via" (or "through"?) because it's an item list, and all items in the list begin with "by..." or "through..." or something like that.

Reply:
Will fix to use "by" in each case to be uniform